### ISSUE TITLE
request: Flag Action-less File module tasks

### ISSUE THREAD

**Issue by therealbill** [ansible/ansible-lint#459]
# Issue Type
- Feature request

# Desired Behaviour

The Ansible file module can be given a path key but also needs an action to take on it. It would be beneficial for ansible-lint to catch it when someone forgets that and does something like this:

```yaml
- name: copy some file into place
  file:
     path: /opt/some/place/myfile
```

The above example will cause an error in an ansible run (absent file) if the file doesn't exist, and do nothing if it does. So it seems to me that the linter could check to make sure the file section contains at least one valid directive beyond "path" and flag that as an error, when the path key is used.

**Comment by ssbarnea** [comment]
Closing as it seems of very little practical use. If someone is creating a PR, I will review it.
